src/includes/configuration/auto-session-tracking/android.mdx

```diff
@@ -1,3 +1,5 @@
+To benefit from the health data, you must use at least version 2.1.0 of the Android SDK, and enable the collection of release health metrics in the AndroidManifest.xml file. Release health is captured, by default, in our Android version 3.0 and higher, unless you specifically disabled collecting it in the initialization options for the SDK.
```

Comment:
```suggestion
To benefit from the health data, you must use at least version 2.1.0 of the Android SDK, and enable the collection of release health metrics in the initialization options for the SDK. Release health is captured, by default, in our Android version 3.0 and higher, unless you specifically disabled collecting it in the initialization options for the SDK.
```

Reply:
This will work, unless we'd like to not mention 2.1.0, and focus only on 3.0. Let me know.